app/javascript/crayons/Button/Button.jsx

```diff
@@ -124,6 +124,7 @@ Button.defaultProps = {
   tagName: 'button',
   size: 'default',
   contentType: 'text',
+  variant: undefined,
```

Comment:
given line 45 I wonder if variant shouldn't default to `primary`, let me know what you think!

Reply:
It was expecting a default value but since it wasn't passed I thought to make it undefined as passing ```js variant: ''``` was throwing error. If you want then I'll pass 'primary' as default.